internal/impl/kafka/enterprise/integration_test.go

```diff
@@ -35,6 +35,7 @@ import (
 	_ "github.com/redpanda-data/benthos/v4/public/components/pure"
 	"github.com/redpanda-data/benthos/v4/public/service"
 	"github.com/redpanda-data/benthos/v4/public/service/integration"
+
```

Comment:
Guess we can get rid of this change

Reply:
`make fmt` keeps adding that.

Reply:
Oh, oops… Sounds like our linter is missing some stuff. Thanks for catching it!